tests/client_test.toit

```diff
@@ -0,0 +1,137 @@
+// Copyright (C) 2022 Toitware ApS.
```

Comment:
Just a renamed version of the `simple_client_test.toit`.
The `simple_client_test.toit` should be considered a new test.